conflictmonitor/docker/mongo/b_create_indexes.js

```diff
@@ -0,0 +1,618 @@
+// Create indexes on all collections
+
+/*
+This is the second script responsible for configuring mongoDB automatically on startup.
+This script is responsible for creating users, creating collections, adding indexes, and configuring TTLs
+For more information see the header in a_init_replicas.js
+*/
+
+console.log('')
```

Comment:
Needed?

Reply:
Is this referring to the entire file, or the prints?

Reply:
> Is this referring to the entire file, or the prints?

Probably the log statement

Reply:
Definitely the empty log console.log('')

Reply:
Yes, just the empty log statement. Not sure what it was added in here.

Reply:
Removed!